app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -215,31 +240,45 @@ def payments
   end
 
   def orders_and_fulfillment
-    # -- Prepare Date Params
-    prepare_date_params params
+    if request.format.json?
+      # -- Prepare Date Params
+      prepare_date_params params
+      @report = OpenFoodNetwork::OrdersAndFulfillmentsReport.new spree_current_user, params
+
+      report_data = {
+        line_items: @report.line_items_serialized,
+        orders: @report.orders_serialized,
+        products: @report.products_serialized,
+        variants: @report.variants_serialized }
+      render json: report_data
```

Comment:
It would be cool to use the `OrdersAndFulfillmentSerializer` here:

````ruby
render json: @report, serializer: Api::Admin::Reports::OrdersAndFulfillmentSerializer
````

Then you could remove the `line_items_serialized`, `orders_serialized` methods from the report, and the `report_data` hash here.  I think this is what I was intending to do in my spike, but I never got around to it.